roles/lib_openshift/src/ansible/oc_sdnvalidator.py

```diff
@@ -0,0 +1,24 @@
+# pylint: skip-file
+# flake8: noqa
+
+def main():
+    '''
+    ansible oc module for validating OpenShift SDN objects
+    '''
+
+    module = AnsibleModule(
+        argument_spec=dict(
+            kubeconfig=dict(default='/etc/origin/master/admin.kubeconfig', type='str'),
```

Comment:
There is a differentiation based on product name and version, as of 3.1 or 3.2 (I can't remember) everything standardized on 'etc/origin'. That said, if using `oc cluster up` or the images directly, the location will be different.